Sources/Turf/Simplifier.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+
+enum Simplifier {
```

Comment:
Interesting use of an enumeration as an effective namespace, rather than a struct that’s more commonly used for this purpose but comes with initializers. This pattern might be worth following elsewhere in the codebase.